src/frontend/src/components/NavBarV2/index.module.scss

```diff
@@ -0,0 +1,101 @@
+@import "src/common/styles/colors";
```

Comment:
I noticed we have both scss and ts styles for this component. Why did that happen?